NightscoutServiceKit/NightscoutService.swift

```diff
@@ -90,6 +90,8 @@ public final class NightscoutService: Service {
 
     public func completeOnboard() {
         isOnboarded = true
+
+        saveCredentials()
```

Comment:
Hm.  Not sure I am familiar with the NightscoutService enough to approve...I guess it's ok?

Reply:
This was just fixing an issue when the service was created, but not yet onboarded and the app was quit/restarted. This ensures the credentials are actually saved.